src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -22,8 +24,14 @@ public void actionPerformed(ActionEvent e) {
 		if(selectedFigure.equals("figure joe 1"))
 		{
 			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-		}else {
+		}else if(selectedFigure.equals("figure joe 1")){
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+		}else if(selectedFigure.equals("factory triangle")) {
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.